docs/modules/ROOT/pages/cops_naming.adoc

```diff
@@ -381,9 +381,13 @@ anything/using_snake_case.rake
 | `true`
 | Boolean
 
+| CheckDefinitionPathHierarchyRoots
```

Comment:
The cop doc files are generated automatically so should not be included here.

Reply:
how are they generated ? ... on merge or I have to run something ?

Reply:
They’re generated as part of the release process.